### PR TITLE
Fix developer warnings

### LINKED ISSUE #10002: Identify and fix all developer console warnings and errors in Meshery UI
### Current Behavior
Presently there are so many warnings and errors coming in developer console, most of them are because of passing classNames and attributes in the wrong way.

### Expected Behavior
Developer console should contain fewer warnings and errors, which helps in to debug better

### Screenshots/Logs
Screenshot 2024-01-18 at 12 55 00 AM

### Acceptance test
- [ ] Identify all errors and write up in PR.
- [ ] Fixing these errors should not change the present behavior of UI.
- [ ] Remove all console.logs

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by JyothiSwaroopReddy07**
**Notes for Reviewers**

This PR fixes #10002

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b7506a50a5a0db64e78e02b50a21dcca8cd90d86 at: https://65ab787ec319e862e0ad6669--meshery-docs-preview.netlify.app

File: docs/pages/reference/rest-api-reference.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Each of Meshery's APIs are subject to the following authentication and authoriza
...
+  Requests to any of the API endpoints must be authenticated and include a valid JWT access token in the HTTP headers. Type of authentication is determined by the selected <a href='/extensibility/providers'>Providers</a>. Use of the Local Provider, "None", puts Meshery into single-user mode and does not require authentication. {% include alert.html type="dark" title="What are authentication tokens?" content="Meshery authentication tokens allow users or systems to authenticate with Meshery Server via either its two clients, <a href='/reference/mesheryctl'>Meshery CLI</a> and <a href='/extensibility/api#how-to-get-your-token'>Meshery UI</a>, or its two APIs: <a href='/reference/rest-apis'>REST</a> or <a href='/reference/graphql-apis'>GraphQL</a>. <p>Meshery's authentication token system provide secure access to Meshery's powerful management features.</p>" %}
```
Comment (leecalcote) [inline docs/pages/reference/rest-api-reference.md:18]: This is a good change / fix.

**Review by leecalcote — COMMENTED** [review]
@meshery/maintainers @meshery/meshery-ui-maintainers do we want all console.logs removed?

**Comment by JyothiSwaroopReddy07** [comment]
> @meshery/maintainers @meshery/meshery-ui-maintainers do we want all console.logs removed?

that commit is for another PR in next issue, I have created new branch. but the commit is being added to this pull request

**Comment by JyothiSwaroopReddy07** [comment]
https://github.com/meshery/meshery/pull/10027

this was the pr for that issue.

**Review by aabidsofi19 — APPROVED** [review]

**Comment by sudhanshutech** [comment]
> @meshery/maintainers @meshery/meshery-ui-maintainers do we want all console.logs removed?

removing does not do any harm i guess and we are exposing many unnecessary info to users and will just create noise in logs

**Review by leecalcote — APPROVED** [review]
Thank you, gents. 👏

**Pull request merged**